cmd/titus-storage/mount.go

```diff
@@ -44,10 +48,88 @@ func mountIt(ctx context.Context, dev string, fstype string, mountPoint string,
 	}
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	l.Printf("%s %s", strings.Join(cmd.Env, " "), mountCommand)
+	l.Printf("%s %s", strings.Join(cmd.Env, " "), mountBlockDeviceCommand)
 	return cmd.Run()
 }
 
+func bindMountInContainer(ctx context.Context, source string, target string, mountPerm string, pid1Dir string) error {
```

Comment:
I strongly recommend you make this a struct because it'll be harder to keep track of args.

Reply:
Great idea. Let me get https://github.com/Netflix/titus-executor/pull/586 merged and then I'll rebase and use the struct defined there.